invenio_rdm_migrator/load/postgresql/transactions/execute.py

```diff
@@ -76,9 +76,10 @@ def _load(self, transaction_group):
                     session.flush()
                 except Exception:
                     logger.exception(
-                        f"Could not {transaction_group['tx_id']} ({transaction_group['tx_group_id']})",
+                        f"Could not {tx['tx_id']} ({tx['action']})",
                         exc_info=1,
                     )
                     session.rollback()
+                    return
```

Comment:
```suggestion
                    raise
```

minor: wouldn't we actually want to completely bail in this case? if a transaction fails to go in, that means that we'll be in an inconsistent state and won't be able to receive any future things

Reply:
It really depends how we want to deal with it.
a) `return`, it means this transaction (and subsequent ones on the data they touch) will fail or be inconsistent. We would need to implement a way to track this.
b) `raise`, which at the moment would go up and finally stop the stream. It would be safer, but it means we would need to manually resume, set offsets, etc. 

I agree b) is better at the moment, so I'm changing it, but we should evaluate a)

Reply:
I think `b)` is the safest, otherwise we might end up with very bad data consistency issues (imagine e.g. a new user signs up and uploads 10 records, and the user creation fails).

In general, I think the philosophy we should have for the tool should be:

- When running in development/QA mode, we just log transformation errors that might affect "data quality"
- When running the real-deal/production, we immediately fail when if anything goes wrong. I think it's very unlikely that the issue is transient (i.e. restarting won't change anything).

Reply:
Agree!